-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-47822/44432

ERROR: Build errors found during clang-tidy run.
```
src/CondCore/ESSources/plugins/CondDBESSource.cc:762:12: error: no matching member function for call to 'addObsoleteUntracked' [clang-diagnostic-error]
  762 |   dbParams.addObsoleteUntracked<std::string>("transactionId", "")->setComment("This parameter is not strictly needed by PoolDBESSource, but the WMCore infrastructure requires it. Candidate for deletion");
      |   ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02884/el8_amd64_gcc12/cms/cmssw/CMSSW_15_1_X_2025-04-08-2300/src/FWCore/ParameterSet/interface/ParameterSetDescription.h:151:31: note: candidate function template not viable: requires single argument 'iLabel', but 2 arguments were provided
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/CondCore/ESSources/plugins/CondDBESSource.cc
+++ b/CondCore/ESSources/plugins/CondDBESSource.cc
@@ -759,6 +759,10 @@ void CondDBESSource::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   dbParams.addUntracked<std::string>("security", "");
   dbParams.addUntracked<int>("messageLevel", 0);
   dbParams.addUntracked<int>("connectionTimeout", 0);
+  dbParams.addObsoleteUntracked<std::string>("transactionId")
+      ->setComment(
+          "This parameter is not strictly needed by PoolDBESSource, but the WMCore infrastructure requires it. "
+          "Candidate for deletion");
   desc.add("DBParameters", dbParams);
 
   desc.add<std::string>("connect", std::string("frontier://FrontierProd/CMS_CONDITIONS"));

